pkg/core/native/native_nep17.go

```diff
@@ -173,11 +173,12 @@ func (c *nep17TokenNative) updateAccBalance(ic *interop.Context, acc util.Uint16
 			return errors.New("insufficient funds")
 		}
 		si = state.StorageItem{}
-	} else if amount.Sign() == 0 && requiredBalance != nil {
+	} else if amount.Sign() <= 0 && requiredBalance != nil {
```

Comment:
Ugh...

Reply:
Maybe it's better to put back `si` after receiving an error from `incBalance`? Otherwise we're unwrapping/comparing the balance twice.

Reply:
What about this though?

Reply:
Fixed, forgot to push.